packages/camera/camera_avfoundation/example/ios/RunnerTests/CameraSettingsTests.m

```diff
@@ -0,0 +1,256 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+@import camera_avfoundation;
+@import camera_avfoundation.Test;
+@import XCTest;
+@import AVFoundation;
+#import <OCMock/OCMock.h>
+#import "CameraTestUtils.h"
+#import "MockFLTThreadSafeFlutterResult.h"
+
+static const char *TEST_RESOLUTION_PRESET = "medium";
+static const int TEST_FPS = 15;
+static const int TEST_VIDEO_BITRATE = 200000;
+static const int TEST_AUDIO_BITRATE = 32000;
+static const bool TEST_ENABLE_AUDIO = YES;
+
+@interface CameraSettingsTests : XCTestCase
+@property(readonly, nonatomic) FLTCam *camera;
```

Comment:
Test fixtures should avoid being stateful whenever possible. Why is this needed?

Reply:
Just copied from another test... Moved to test body.